internal/pods/schedule.go

```diff
@@ -270,3 +285,35 @@ func addNodeSelectorTerm(nodeSelTerms []v1.NodeSelectorTerm, label string,
 		},
 	}})
 }
+
+func checkNSLabels(nsLabels map[string]string) {
+	expectedLabels := map[string]string{
+		"pod-security.kubernetes.io/enforce": "privileged",
+		"pod-security.kubernetes.io/audit":   "privileged",
+		"pod-security.kubernetes.io/warn":    "privileged",
+	}
+
+	missingLabels := map[string]string{}
+
+	for key := range expectedLabels {
+		val, found := nsLabels[key]
+		if !found || val != "privileged" {
+			missingLabels[key] = "privileged"
+		}
+	}
+
+	if len(missingLabels) != 0 {
+		warnAbout(missingLabels)
+	}
+}
+
+func warnAbout(missingLabels map[string]string) {
+	status := cli.NewReporter()
+	status.Warning("Kubernetes 1.23/OpenShift Container Platform 4.11 and above expects namespaces to have security labels as required by" +
+		" `PodSecurity Admission controller`. Otherwise, the related warnings would be logged. To avoid such warnings, make sure the" +
+		" namespace specified has below labels:")
```

Comment:
For upstream Submariner, reduce the message length by only mentioning K8s, and indicate that the warnings are innocuous:

```suggestion
	status.Warning("Kubernetes 1.23 and above, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see warnings in subctl's output. subctl should work fine, but you can avoid the warnings and ensure correct behavior by adding these labels to the namespace:")
```

(unwrapped). While we’re at it, please specify the namespace in the output — the user just specified it but it’s always best to have fully actionable error messages.